discovery/syncer.go

```diff
@@ -52,6 +53,17 @@ const (
 	chansSynced
 )
 
+const (
+	// DefaultMaxUndelayedQueryReplies specifies how many gossip queries we
+	// will respond to immediately before starting to delay responses.
+	DefaultMaxUndelayedQueryReplies = 10
```

Comment:
The initial setup requires 1 reply from each peer, after that the number of queries depends on the chunk size used by each peer. We _should_ be able to fit all sids in one more query since the graph only has 4k channels. Though if the remote peer uses a smaller chunk size, we may delay their time-to initial sync. 

I'm happy to reduce this further, 10 was just for some leeway. To be clear, are you suggesting we make it one or two?

Looking at all the implementations, we can assume that most will query with chunks of roughly 7-12k

Reply:
I initially thought we would just have to reply once for good peers. But as we might expect even good peers to query more than once, I think 5-10 is ok.